lib/airbrake/rack/notice_builder.rb

```diff
@@ -62,7 +62,7 @@ def add_context(notice)
           context[:action] = @controller.action_name
         end
 
-        notice[:context].merge!(@user.to_hash) if @user
+        notice[:context].merge!(@user.data) if @user
```

Comment:
Because this is Ruby, and this is not a Ruby convention. I don't insist, it's minor, but I think `data` is less readable because I have to look-up the method to see what it returns, unlike with `to_hash`.